VS 21 February/VS 21 February/Task 10/Task 10.3 point/Point.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+namespace VS_21_February.Task10.point
+{
+	public class Point <T> where T : struct
+	{
+		public T x;
+		public T y;
+
+		public T X
+		{
+			get { return x; }
+			set { x = value; }
+		}
+
```

Comment:
it this case you can use auto properties
```suggestion
		public T X {get; set;}

```

Reply:
fixed, thanks.